### PR TITLE
Create a Badge component

### PR DESCRIPTION
**Pull request by stevegood**
I've created a simple Badge component based on the implementation examples here: http://materializecss.com/badges.html

### CODE REVIEW

File: src/Badge.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class Badge extends React.Component {
+  render() {
+    let classes = ['badge'];
+    if (this.props.text) {
+      classes.push(this.props.text);
```
Comment (isaiah) [inline src/Badge.js]: Please use `classnames` (https://github.com/JedWatson/classnames) library to compose class names.
I am not sure if the text could be other than `new`, in that case it should be a boolean prop.

Comment (stevegood) [inline src/Badge.js]: Yeah, sorry, I was running on very little sleep,  'new' should definitely be a boolean prop not a string.  I can close the PR , make the changes and submit a new one.

**Comment by stevegood** [comment]
Closing to make additional changes.

File: src/Badge.js (hunk 1 above)
Comment (isaiah) [inline src/Badge.js]: You don't have to close this PR, if you just push your change to your branch it will show here.

**Comment by isaiah** [comment]
Hi @stevegood, if you don't have time to make the change, I'll merge this and fix it myself, it could be a quite useful component.

**Comment by stevegood** [comment]
I have the change made and additionally I added to the docs.  I'm about to submit a new PR for those changes.

**Pull request merged**